src/Controls/tests/DeviceTests/ControlsHandlerTestBase.iOS.cs

```diff
@@ -22,14 +22,15 @@ Task SetupWindowForTests<THandler>(IWindow window, Func<Task> runTests, IMauiCon
 			mauiContext ??= MauiContext;
 			return InvokeOnMainThreadAsync(async () =>
 			{
+				IElementHandler windowHandler = null;
 				try
 				{
-					_ = window.ToHandler(mauiContext);
+					windowHandler = window.ToHandler(mauiContext);
```

Comment:
These changes fix some issues for @jonathanpeppers on 
 this https://github.com/dotnet/maui/pull/13400